sitestats/sitestats-tool/src/webapp/html/pages/ReportsEditPage.html

```diff
@@ -132,7 +132,7 @@ <h4 class="summaryHeader">
                             <td>
                                 <!-- Tool selection -->
                                 <div id="what-tools-select" style="display:none">
-	                               <select wicket:id="reportParams.whatToolIds" style="overflow: auto;" size="7" multiple="multiple" class="what-tools-select-component responsive-form">
+	                               <select wicket:id="reportParams.whatToolIds" style="overflow: auto; font-family: FontAwesome;" size="7" multiple="multiple" class="what-tools-select-component responsive-form">
```

Comment:
Remove this inline styles and put them in an appropriate CSS class instead

Reply:
HTML pages on Sitestats tool are really a disaster about inline CSS, please file a new JIRA for that purpose. Thanks!